test/unit/CMakeLists.txt

```diff
@@ -147,3 +147,12 @@ if(NMODL_ENABLE_PYTHON_BINDINGS)
   set_tests_properties(Pybind PROPERTIES ENVIRONMENT
                                          PYTHONPATH=${PROJECT_BINARY_DIR}/lib:$ENV{PYTHONPATH})
 endif()
+
+# =============================================================================
+# translation of mod files
+# =============================================================================
+file(GLOB modfiles "${NMODL_PROJECT_SOURCE_DIR}/test/integration/mod/*.mod")
+foreach(modfile ${modfiles})
+  get_filename_component(modfile_name "${modfile}" NAME)
+  add_test(NAME ${modfile_name} COMMAND ${PROJECT_BINARY_DIR}/bin/nmodl ${modfile})
+endforeach()
```

Comment:
@kotsaloscv: `nitpick` : As this test is more for integration, may be we could add `CMakeLists.txt` in `test/integration` and run these tests there?